lib/src/signed_video_h26x_auth.c

```diff
@@ -691,6 +691,39 @@ compute_gop_hash(signed_video_t *self, h26x_nalu_list_item_t *sei)
   return status;
 }
 
+/**
+ * Decodes the SEI message, retrieves necessary parameters for authentication, and computes the hash
+ * for authenticity.
+ */
+static svrc_t
+prepare_golden_sei(signed_video_t *self, h26x_nalu_list_item_t *sei)
+{
+  assert(self);
+  svrc_t status = SV_UNKNOWN_FAILURE;
+  sign_or_verify_data_t *verify_data = self->verify_data;
+  const size_t hash_size = verify_data->hash_size;
+  self->gop_state.has_sei = true;
```

Comment:
This is used in `prepare_for_validation`, so move it to the line right before it.